### DISCUSSION TITLE
vehicles disappear

### DISCUSSION THREAD

**Discussion by balcerowicz** [YimMenu/YimMenu#2678] [Q&A] [answered]
when I spawn vehicle and go away from it car will disappear, is it a menu bug or should I switch some option?

**Comment by lonelybud** [comment]
if you spawn vehicle from spawn tab it will disappear it is coded as such

**Reply by balcerowicz** [reply]
but it wasnt like that before

**Reply by lonelybud** [reply]
https://github.com/YimMenu/YimMenu/blob/ad1dfc71d2d5b2a9fb16f1c2009e4bc60bd65231/src/views/vehicle/view_spawn_vehicle.cpp#L197

**Reply by lonelybud** [reply, marked as answer]
present solution is to spawn from persist vehicle
